app/views/error.html.ts

```diff
@@ -57,6 +57,9 @@ let template = compileTemplate(`
 export default class ErrorView extends View {
 
   async render(action: Action, response: ServerResponse, error: any, options: RenderOptions) {
+    // Action must be defined, or lodash's template will throw an error
+    error.action = error.action || null;
```

Comment:
Actually, if we keep the `data.error` prefixes, and use the `variable` option when compiling the template, we can ensure no `with` statement is used, and therefore no reference errors will be thrown:

```
template = compileTemplate(..., { variable: 'data' });
```